### PR TITLE
TEN-70 added audit events for reporting fraud

### PR DESCRIPTION
**Pull request by Chrissystk**

### CODE REVIEW

File: test/uk/gov/hmrc/taxenrolmentassignmentfrontend/controllers/ReportSuspiciousIDControllerSpec.scala [hunk 1]
```diff
@@ -83,6 +100,14 @@ class ReportSuspiciousIDControllerSpec extends TestFixture {
...
         mockGetAccountTypeAndRedirectUrlSuccess(randomAccountType)
 
+        val auditEvent = AuditEvent.auditReportSuspiciousPTAccount(
+          accountDetails
+        )(requestWithAccountType(PT_ASSIGNED_TO_OTHER_USER))
+        (mockAuditHandler
+          .audit(_: AuditEvent)(_: HeaderCarrier))
+          .expects(auditEvent, *)
+          .returning(Future.successful((): Unit))
+
```
Comment (jparekh59) [inline test/uk/gov/hmrc/taxenrolmentassignmentfrontend/controllers/ReportSuspiciousIDControllerSpec.scala:110]: Could you add .once() to eliminate multiple audits not fired.

File: test/uk/gov/hmrc/taxenrolmentassignmentfrontend/controllers/ReportSuspiciousIDControllerSpec.scala [hunk 2]
```diff
@@ -201,6 +226,14 @@ class ReportSuspiciousIDControllerSpec extends TestFixture {
           .returning(createInboundResult(accountDetails))
         mockGetAccountTypeAndRedirectUrlSuccess(randomAccountType)
 
+        val auditEvent = AuditEvent.auditReportSuspiciousSAAccount(
+          accountDetails
+        )(requestWithAccountType(SA_ASSIGNED_TO_OTHER_USER))
+        (mockAuditHandler
+          .audit(_: AuditEvent)(_: HeaderCarrier))
+          .expects(auditEvent, *)
+          .returning(Future.successful((): Unit))
```
Comment (jparekh59) [inline test/uk/gov/hmrc/taxenrolmentassignmentfrontend/controllers/ReportSuspiciousIDControllerSpec.scala:235]: Could you add .once() to eliminate multiple audits not fired.

**Review by jparekh59 — CHANGES_REQUESTED** [review]

File: test/uk/gov/hmrc/taxenrolmentassignmentfrontend/controllers/ReportSuspiciousIDControllerSpec.scala (hunk 1 above)
Comment (Chrissystk) [reply test/uk/gov/hmrc/taxenrolmentassignmentfrontend/controllers/ReportSuspiciousIDControllerSpec.scala:111]: done

File: test/uk/gov/hmrc/taxenrolmentassignmentfrontend/controllers/ReportSuspiciousIDControllerSpec.scala (hunk 2 above)
Comment (Chrissystk) [reply test/uk/gov/hmrc/taxenrolmentassignmentfrontend/controllers/ReportSuspiciousIDControllerSpec.scala:236]: done

**Review by jparekh59 — APPROVED** [review]

**Pull request merged**